e2e/scripts/smoke_test.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash -e
+
+test_source_repo=$(pwd)
+docker_test_repo=/workspace/go/src/github.com/rook/rook
+git_smoke_test_directory=github.com/rook/rook/e2e/tests/smoke
+container_image=quay.io/quantum/rook-test
+tmp_docker_sock_path=/tmp/docker.sock
+results_dir=results
+results_filename=test.log
+
+#create the rook infrastructure container
+rook_infra::create() {
+    export id=$(docker run \
+        --net=host \
+        -d \
+        -e container=docker \
+        -e GOPATH=/workspace/go \
+        -e GOROOT=/usr/lib/go \
+        --privileged \
+        --security-opt=seccomp:unconfined \
+        --cap-add=SYS_ADMIN \
+        -v /dev:/dev \
```

Comment:
are these volume mounting needed? This is to create the rook-infra. Only the k8s nodes need these ceph related volumes.